packages/core/package.json

```diff
@@ -36,6 +36,7 @@
     "@ethersproject/abstract-provider": "^5.7.0",
     "@openzeppelin/hardhat-upgrades": "^1.22.1",
     "@openzeppelin/upgrades-core": "^1.20.6",
+    "chalk": "^4.1.2",
```

Comment:
Allows for nicer console outputs with color and stuff. I use this for the new `chugsplashLog` function.